liquibase-core/src/main/java/liquibase/command/core/InternalExecuteSqlCommandStep.java

```diff
@@ -61,10 +64,16 @@ public void run(CommandResultsBuilder resultsBuilder) throws Exception {
         } else {
             final PathHandlerFactory pathHandlerFactory = Scope.getCurrentScope().getSingleton(PathHandlerFactory.class);
             Resource resource = pathHandlerFactory.getResource(sqlFile);
-            if (!resource.exists()){
-                throw new LiquibaseException(FileUtil.getFileNotFoundMessage(sqlFile));
+            if (!resource.exists()) {
```

Comment:
Similarly to my `CreateProcedure` comment, I would put this logic as a function of `AbstractCommandStep` class as something like this:
```
public String returnEmptyResourceIfMissingFile(String sqlFile) {
    if (ChangeLogParserConfiguration.WARN_ON_MISSING_CHANGELOGS.getCurrentValue()) {
        Scope.getCurrentScope().getLog(getClass()).warning(FileUtil.getFileNotFoundMessage(path));
        return FileUtil.EMPTY_FILE;
    }
    else {
        throw new LiquibaseException(FileUtil.getFileNotFoundMessage(sqlFile));
    }
}
```